lms/templates/widgets/footer-language-selector.html

```diff
@@ -0,0 +1,55 @@
+## Language-selection widget for the footer.
+##
+## Requires settings.LANGUAGE_COOKIE.
+<%page expression_filter="h"/>
+<%!
+  from babel import Locale
+  from django.conf import settings
+  from django.utils.translation import ugettext as _
+
+  from openedx.core.djangoapps.lang_pref import COOKIE_DURATION
+  from openedx.core.djangoapps.lang_pref.api import released_languages
+  from openedx.core.djangolib.js_utils import js_escaped_string
+
+  # Make sure LANGUAGE_COOKIE is present.
+  if not settings.LANGUAGE_COOKIE:
+      raise ValueError('settings.LANGUAGE_COOKIE is required to use footer-language-selector.')
+%>
+
+<div>
+    <label for="footer-language-selector">
+        <i class="fa fa-globe" aria-hidden="true"></i><span class="sr">${_("Choose Language")}</span>
```

Comment:
Using spans for icons is preferred because it's not semantic to use the ```i``` element (which is meant for italicizing text). What isn't working when you change it to a span?